### ISSUE TITLE
The package does not build.

### ISSUE THREAD

**Issue by pveber** [pveber/morse#15]
I still have doc-related errors when I try to build. Please fix the project so that build/check/test/doc generation work as indicated in the [README](README.md) (maybe only the instructions are incorrect?)

**Comment by pveber** [comment]
The problem is not fixed, I still have:
```
pveber@umr5558-johndoe:~/w$ R CMD build morse
* checking for file ‘morse/DESCRIPTION’ ... OK
* preparing ‘morse’:
* checking DESCRIPTION meta-information ... OK
* installing the package to build vignettes
      -----------------------------------
* installing *source* package ‘morse’ ...
** R
** data
** preparing package for lazy loading
** help
*** installing help indices
Erreur dans Rd_info(db[[i]]) : 
  champ \title manquant ou vide dans '/tmp/Rtmpr3vBMJ/Rbuild7806776a21f1/morse/man/morse-package.Rd'
les fichiers Rd doivent avoir un \title non vide
Voir le chapitre 'Writing R documentation' dans le manuel 'Writing R Extensions'.
* removing ‘/tmp/Rtmpr3vBMJ/Rinst78066e4915c7/morse’
      -----------------------------------
ERROR: package installation failed
```
Don't you see the same issue?

**Comment by philipperuiz** [comment]
Because the file `DESCRIPTION` are not up to date

**Comment by pveber** [comment]
Could you take a look at it?

**Comment by pveber** [comment]
fixed in 9c00c24c5b2b
